core/src/test/java/com/google/googlejavaformat/java/MainTest.java

```diff
@@ -251,4 +251,28 @@ public void importRemoveErrorParseError() throws Exception {
     assertThat(main.format("-")).isEqualTo(1);
     assertThat(err.toString()).contains("<stdin>:4:3: error: class, interface, or enum expected");
   }
+
+  @Test
+  public void validate() throws Exception {
+    Main main =
+        new Main(
+            new PrintWriter(System.out, true),
+            new PrintWriter(System.err, true),
+            System.in);
+    int errorCodeOk = main.format("--validate", //
```

Comment:
Oh, it's also not clear to me why you included blank comments at the end of this line and a few of the lines below. Could you explain why they're there? Are they even needed?